integration_tests/pages/apply/risks-and-needs/utils.ts

```diff
@@ -2,6 +2,13 @@ export const pageIsActiveInNavigation = (linkText: string): void => {
   cy.get('.moj-side-navigation__item--active a').contains(linkText)
 }
 
+export const pageHasLinkToGuidance = (): void => {
+  cy.get('a')
+    .contains('Who do I need to contact for health care information?')
+    .should('have.attr', 'href')
+    .and('match', new RegExp(Cypress._.escapeRegExp('tasks/health-needs/pages/guidance')))
```

Comment:
I see (from your previous work) that you can pass in the value as a third argument to the should() method - https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/blob/main/integration_tests/pages/apply/ineligiblePage.ts#L16

Reply:
well spotted -- I was keen to try the "chaining"!